packages/commonwealth/client/scripts/views/components/sidebar/CollapsableSidebarButton/CollapsableSidebarButton.scss

```diff
@@ -0,0 +1,72 @@
+.CollapsableSidebarButtonUnscoped {
+  // This is an invisible box that allows us to hover over it in order to make the collapsable button visible
+  .hover-box {
+    position: fixed;
+    height: 100vh;
```

Comment:
what is expected behaviour? Looking at this line, I would assume that we should hover over the whole edge of the browser and the arrows would appear. However, right now, icon shows up only on icon hover. Not bad for me, but the code does not seem to be doing what it looks like to do

https://github.com/hicommonwealth/commonwealth/assets/14819225/2cef657e-c0a1-48dd-990f-603d9f60ed53